tests/foreman/api/test_classparameters.py

```diff
@@ -184,13 +324,26 @@ def test_negative_update_parameter_type(self):
 
         1. Parameter not updated with string type for invalid value.
         2. Error raised for invalid default value.
-
-        @caseautomation: notautomated
         """
+        sc_param = self.sc_params_list.pop()
+        for test_data in invalid_sc_parameters_data():
+            with self.subTest(test_data):
+                with self.assertRaises(HTTPError) as context:
```

Comment:
nitpick: as we now support python 2.7+, you may use following syntax:
```python
with A() as X, B() as Y, C() as Z:
    do_something()
```
to avoid nested context managers.